cypress/e2e/pages/onDemandTV/index.cy.js

```diff
@@ -1,33 +1,472 @@
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/telugu/bbc_telugu_tv/tv/w172xtxyt9k8y6p', // On Demand Episode
+    service: 'telugu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/telugu/bbc_telugu_tv/tv/w172xtxyt9k8y6p.lite', // On Demand Episode
+    service: 'telugu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv_programmes/w13xttp9', // Brand
+    service: 'ukrainian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv_programmes/w13xttp9.lite', // Brand
+    service: 'ukrainian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv/w172xtvv0w8tq9m', // Episode
+    service: 'ukrainian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv/w172xtvv0w8tq9m.lite', // Episode
+    service: 'ukrainian',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv/w172xct4hclz27g', // Episode
+    service: 'ukrainian',
+    runforEnv: 'local',
+    tests,
+  },
+  {
+    path: '/ukrainian/bbc_ukrainian_tv/tv/w172xct4hclz27g.lite', // Episode
+    service: 'ukrainian',
+    runforEnv: 'local',
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv_programmes/w13xttn1', // Brand
+    service: 'urdu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv_programmes/w13xttn1.lite', // Brand
+    service: 'urdu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv/w172xtwfxsl890n', // Episode
+    service: 'urdu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv/w172xtwfxsl890n.lite', // Episode
+    service: 'urdu',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv/w172xtwkxr1xtlz', // Episode
+    service: 'urdu',
+    runforEnv: 'local',
+    tests,
+  },
+  {
+    path: '/urdu/bbc_urdu_tv/tv/w172xtwkxr1xtlz.lite', // Episode
+    service: 'urdu',
+    runforEnv: 'local',
+    tests,
+  },
+  {
+    path: '/uzbek/bbc_uzbek_tv/tv_programmes/w13xttqv', // Brand
+    service: 'uzbek',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/uzbek/bbc_uzbek_tv/tv_programmes/w13xttqv.lite', // Brand
+    service: 'uzbek',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/uzbek/bbc_uzbek_tv/tv/w172xtx1tpwq8tz', // Episode
+    service: 'uzbek',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/uzbek/bbc_uzbek_tv/tv/w172xtx1tpwq8tz.lite', // Episode
+    service: 'uzbek',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+];
+
+const liteTestSuites = testSuites.map(testSuite => {
+  return {
+    ...testSuite,
+    path: `${testSuite.path}.lite`,
+  };
+});
+
+runTestsForPage({
+  pageType,
+  testSuites,
+  liteTestSuites,
```

Comment:
```suggestion
  testSuites
```

or if you decide to keep `liteTestSuites`

```suggestion
  testSuites: [...testSuites, ...liteTestSuites]
```